### DISCUSSION TITLE
很奇怪的一个评论排序问题 || very strange comment sorting problem

### DISCUSSION THREAD

**Discussion by ayakasuki** [walinejs/waline#2031] [General]
我看了看帖子说1.21修复了
但是现在1.31.2重新复现这个问题
![image](https://github.com/walinejs/waline/assets/20859971/5b91cd2e-49b5-4eaa-ac38-f4643f0a48c7)
![image](https://github.com/walinejs/waline/assets/20859971/9b1a7dcf-e290-4f9f-9b0f-a48c49db33a1)
![image](https://github.com/walinejs/waline/assets/20859971/7b9cc499-bbfd-44ec-bc94-4d9de98f8e09)
三个按钮直接出现不同的评论 而不会完全显示完 很怪。。。

---
I looked at the post and said 1.21 fixed it
But now 1.31.2 reproduces this problem
![image](https://github.com/walinejs/waline/assets/20859971/5b91cd2e-49b5-4eaa-ac38-f4643f0a48c7)
![image](https://github.com/walinejs/waline/assets/20859971/9b1a7dcf-e290-4f9f-9b0f-a48c49db33a1)
![image](https://github.com/walinejs/waline/assets/20859971/7b9cc499-bbfd-44ec-bc94-4d9de98f8e09)
It's weird that different comments appear directly on the three buttons without being completely displayed. . .

**Comment by lizheming** [comment]
你这个和之前修复的不是一个问题，可以提供下地址


---
Your problem is not the same as the one you fixed before, you can provide the following address

**Comment by ayakasuki** [comment]
ayakasuki.com



| |
***@***.***
|
|
***@***.***
|




---- 回复的原邮件 ----
| 发件人 | Austin ***@***.***> |
| 日期 | 2023年07月22日 21:42 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [walinejs/waline] 很奇怪的一个评论排序问题 || very strange comment sorting problem (Discussion #2031) |

提供下地址

**Reply by lizheming** [reply]
提供可复现的地址，你的博客就一篇文章和一个关于页可以留言，并没有看到你说的问题。另外再提供下部署方式，我看你这个好像不是正常的部署方式。

另外，建议以后提 dicussion/issue 的时候能够按照官方模板来，如果是问题反馈可以按照官方模板来，如果你觉得是 Bug 建议提 issue，如果要提 discussion 可以提 QA 分类。能在提问题的时候就提供好充足的信息，这样排查问题会更高效一点。

---
Provide a reproducible address, your blog can leave a message on an article and an about page, and I didn't see the problem you mentioned. In addition, the deployment method will be provided. I think this is not a normal deployment method.

In addition, it is recommended to follow the official template when raising a dicussion/issue in the future. If it is a problem feedback, you can follow the official template. If you think it is a bug, it is recommended to raise an issue. If you want to raise a discussion, you can raise the QA category. If you can provide sufficient information when you ask a question, it will be more efficient to troubleshoot the problem.
